### DISCUSSION TITLE
Installation monitor

### DISCUSSION THREAD

**Discussion by easygoer33** [Klocman/Bulk-Crap-Uninstaller#597] [Q&A]
I just started using BCU. I could not find any documentation on installation monitoring. Maybe I am behind the times or just don't understand how BCU is built but I thought the most sure way of uninstalling an app would be to compare the system state (folders, registry etc.) before and after the installation of a program and then delete anything leftover. Then there wouldn't be any doubt about which registry keys to remove either.  Is this not a function in BCU? Any help would be greatly appreciated.

**Comment by Klocman** [comment]
See https://github.com/Klocman/Bulk-Crap-Uninstaller/issues/258
